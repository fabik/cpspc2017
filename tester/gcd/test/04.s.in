10
11 14 19 23 25 26 27 28 29 30
28 29 29 30 30 