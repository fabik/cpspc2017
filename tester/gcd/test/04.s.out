1
31
60
88
115
141
166
189
208
232
